src/lib/Libnet/net_server.c

```diff
@@ -1035,6 +1019,25 @@ init_poll_context(void)
 		log_err(errno, __func__, "could not initialize poll_context");
 		return (-1);
 	}
+#ifdef WIN32
+	/* set a dummy fd in the read set so that	*/
+	/* select() does not return WSAEINVAL 		*/
+	sd_dummy = socket(AF_INET, SOCK_STREAM, 0);
+	if (sd_dummy < 0) {
+		errno = WSAGetLastError();
+		log_err(errno, __func__, "socket() failed");
+		return -1;
+	}
+	if((tpp_em_add_fd(poll_context, sd_dummy, EM_IN) == -1)){
+		int err = errno;
+		snprintf(logbuf, sizeof(logbuf), 
+			"Could not add socket %d to the read set", sd_dummy);
+		log_err(err, __func__, logbuf);
+		close_conn(sd_dummy);
```

Comment:
I think we cannot call close_conn() here since we had not yet done a add_conn(), so better to simply close the socket and return. So replace with CLOSESOCKET(sd_dummy);

Reply:
I've changed it.